packages/phone-number-privacy/src/config.ts

```diff
@@ -1,4 +1,57 @@
-const functions = require('firebase-functions')
-export const functionConfig = functions.config()
+import * as functions from 'firebase-functions'
+import knex from 'knex'
 
-export const PHONE_NUMBER_PRIVACY_SECRET_KEY = functionConfig.envs.secret_key
+const DEV_MODE = process.env.NODE_ENV !== 'production' || process.env.FUNCTIONS_EMULATOR === 'true'
+
+interface Config {
+  salt: {
+    key: string
+  }
+  db: {
+    user: string
+    password: string
+    database: string
+    host: string
+  }
+}
+
+let config: Config
+
+if (DEV_MODE) {
```

Comment:
What is the general strategy for supporting unit tests? Should we create an in-mem mock of the database? Should we run Postgres locally?

Reply:
I'm for in-mem db mock. WDYT?

Reply:
I think that's probably best. The other option is to allow each of us to setup a dedicated postgres for unit testing and use an .env file for configuring the connection string. I think that the interface for us is straightforward enough for us to mock it properly and it forces us to think through the different errors that Postgres can throw.